source/jsonnet/england-wales/individual/blocks/personal-details/term_time_location.jsonnet

```diff
@@ -182,6 +182,18 @@ local otherNonUkAddressOptions = {
     },
   ],
   routing_rules: [
+    {
+      goto: {
+        section: 'End',
```

Comment:
This is currently routing me all the way through to summary, where it should be 'country-of-birth' according to card

Reply:
I am currently checking this with Donna. The `country-of-birth` block is the first block in the next section, so this would be an invalid routing rule.

Reply:
Can confirm this is the correct routing

Reply:
Ok, the card still says otherwise.

Reply:
Cards been updated